docs/02_cleos/03_command-reference/push/push-action.md

```diff
@@ -31,3 +29,65 @@ Push a transaction with a single action
 `--max-net-usage` _UINT_ - set an upper limit on the net usage budget, in bytes, for the transaction (defaults to 0 which means no limit)
 
 `--delay-sec` _UINT_ - set the delay_sec seconds, defaults to 0s
+
+## Output
+
+For actions that return a value `cleos` will have an extra line which will be prefixed with `=>`.
+
+Below you can see three examples of pushing actions `rstring` and `ruint` which return the value passed as input parameter to them:
+
+```shell
+> cleos push action eosio rstring '{"str":null}'  -p eosio@active
+
+executed transaction: 1e72a3ef1fa01a9c90b870fe86bf31413c6a2f40a2722ca72d9dd707f58851af  96 bytes  116 us
+#         eosio <= eosio::rstring               {"str":""}
+=>                                return value: ""
+
+> cleos push action eosio rstring '{"str":"test"}'  -p eosio@active
+
+executed transaction: 2484ba021683bb3c6daaabe291ba291e2689d1be899a293d0a09cc68cf0967fb  96 bytes  116 us
+#         eosio <= eosio::rstring               {"str":"test"}
+=>                                return value: "test"
+
+> cleos push action eosio ruint '{"i":42}'  -p eosio@active
+
+executed transaction: 12c51fd27fad9bb0fa959037a72093ad0b168f5846e916a0e9a295c8416bdf9f  96 bytes  138 us
+#         eosio <= eosio::ruint                 {"i":42}
+=>                                return value: 42
+```
+
+It can also work with C++ structures or classes as well. For a C++ struct like this:
+
+```cpp
+struct tmp {
+    uint32_t id = 1;
+    std::vector<uint8_t> list{ 1, 2 };
+
+    EOSLIB_SERIALIZE(tmp, (id)(list))
+};
+```
+
+and the action `rtmp` returning a default tmp object, output will look like this:
+
+```shell
+> cleos push action eosio rtmp '{}'  -p eosio@active
+
+executed transaction: 42386687d94695d77d67cf901cf8d1a4f83ac9f89382f0b01f3fb51267cc21cd  96 bytes  110 us
+#         eosio <= eosio::rtmp                  ""
+=>                                return value: {"id":1,"list":[1,2]}
+```
+
+When it will not be possible to decode the return value, output will look like this:
```

Comment:
_> When it will not be possible to decode the return value_
@iamveritas You can add an example: ABI doesn't contain information about an action return value.

Reply:
I will change, thank you.